lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +23,161 @@ fields:
+    description: The subject of the questionnaire response. This could be a patient, organization, practitioner, device, etc. This is who/what the answers apply to, but is not necessarily the source of information.
+    action: update
+    example: |
+      {
+        "subject": {
+          "reference": "Patient/12508041"
+        }
+      }
+
+  - name: authored
+    required: 'No'
+    type: dateTime
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.authored
+    description: The date and/or time that this set of answers were last changed.
+    action: update
+    example: |
+      {
+        "authored": "2019-10-25T14:57:43Z"
+      }
+
+  - name: item
+    required: 'Yes'
+    type: List of BackboneElement
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item
+    description: A group or question item from the original questionnaire for which answers are provided. Only answer value types of String, Quantity, and Coding are supported.
+    action: update
+    example: |
+      {
+        "item": [
+          {
+            "linkId": "93",
+            "text": "Tobacco",
+            "item": [
+              {
+                "linkId": "93-123",
+                "text": "Number of Years:",
+                "answer": [
+                  {
+                    "valueQuantity": {
+                      "value": 20
+                    }
+                  }
+                ]
+              }
+            ] 
+          }
+        ]
+      }
+
+  - name: item.linkId
+    required: 'Yes'
+    type: string
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId
+    description: The item from the Questionnaire that corresponds to this item in the QuestionnaireResponse resource.
+    action: update
+    example: |
+      {
+        "linkId": "93"
+      }
+
+  - name: item.text
+    required: 'No'
+    type: string
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.text
+    description: Text that is displayed above the contents of the group or as the text of the question being answered.
+    action: update
+    example: |
+      {
+        "text": "Tobacco"
+      }
+
+  - name: item.item
+    required: 'No'
+    type: List of BackboneElement
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.item
+    description: Questions or sub-groups nested beneath a question or group.
+    action: update
+    example: |
+      {
+        "item": [
+          {
+            "linkId": "93-123",
+            "text": "Number of years:",
+            "answer": [
+              {
+                "valueQuantity": {
+                  "value": 20
+                }
+              }
+            ]
+          }
+        ]
+      }
+
+  - name: item.item.answer
```

Comment:
if we're giving details on the item.item.answer, should we also for the item.item.text and item.item.linkid? maybe not necessary? 

But also, the item.text and item.item.text fields dont actually get written so do we include a section for them or not?

Reply:
We've removed the text field: 6b1e581f4f01eb2d8a1a69a8a0f857da78c0fcc3

I'm not sure about item.item.linkId, what do you think @MadhurRaj ?

Reply:
I went ahead and added the item.item.linkId field, I think it's good to see the inner linkId structure since it's different from item.linkId: https://github.com/cerner/fhir.cerner.com/pull/669/commits/6187215fc0c39f9271ae2977a73fb6475d6efb80#diff-3281c7aca109d3b855df76338281623f6487c08c2d80fd2ae927e480fd58fb8eR150

I don't think we need item.item.text